README.md

```diff
@@ -1784,6 +1784,17 @@ The following extractors use this feature:
 * `po_token`:  Proof of Origin (PO) Token(s) to use. Comma seperated list of PO Tokens in the format `CLIENT.CONTEXT+PO_TOKEN`, e.g. `youtube:po_token=web.gvs+XXX,web.player=XXX,web_safari.gvs+YYY`. Context can be either `gvs` (Google Video Server URLs) or `player` (Innertube player request)
 * `player_js_variant`: The player javascript variant to use for signature and nsig deciphering. The known variants are: `main`, `tce`, `tv`, `tv_es6`, `phone`, `tablet`. Only `main` is recommended as a possible workaround; the others are for debugging purposes. The default is to use what is prescribed by the site, and can be selected with `actual`
 
+##### PO Token settings
+* `po_token`:  Proof of Origin (PO) Token(s) to use. Comma seperated list of PO Tokens in the format `CLIENT.CONTEXT+PO_TOKEN`, e.g. `youtube:po_token=web.gvs+XXX,web.player=XXX,web_safari.gvs+YYY`. Context can be either `gvs` (Google Video Server URLs) or `player` (Innertube player request)
+* `pot_debug`: Enable debug logging for PO Token fetching. Either `true` or `false` (default `false`)
```

Comment:
Wouldn't debug levels (0,1,2 etc) be more useful? `trace` could have a level passed to it as arg. Also, When trace is on, we should force `debug` for POT even without `-v`

Reply:
Hmm, so as to say somewhat decouple it from YDL logger, but have -v also change the log level of it?

Reply:
I'll add a log level for the pot logger. However it is a subset of the YDL/IE logger level, so even if you enable POT debug, if you don't pass -v it will not show.

Reply:
I've reverted this extractor arg back to a true/false option as I don't think there is much use for the different log levels given its behind -v and it's only for debugging. However I've kept underlying implementation based on the levels.